modules/genotypes.nf

```diff
@@ -14,12 +14,23 @@ process filterBED{
 
     script:
         prefix = bedfiles[0].toString().minus('.bed')
-        """
-        TEMPD=\$(mktemp -d)
-        JULIA_DEPOT_PATH=\$TEMPD:/opt julia --project=/TargeneCore.jl --startup-file=no /TargeneCore.jl/bin/prepare_confounders.jl \
-        --input $prefix --output filtered.$prefix --qcfile $qcfile --maf-threshold $params.MAF_THRESHOLD --ld-blocks $ld_blocks --traits $traits filter
-        """
-
+        
```

Comment:
Unless I am missing something, it is not necessary to duplicate the full script. The strategy that I have used so far is to declare a variable containing the option string or an empty string and include that variable in the script, see for instance: https://github.com/TARGENE/targene-pipeline/blob/a637cca5faf56fd0f9a539a492a57755ad2a447e/modules/tmle.nf#L36

Reply:
This should now be updated to a cleaner version: [20d8bc9](https://github.com/TARGENE/targene-pipeline/commit/20d8bc9ef4d20790d5d9a8d91d03e2ce526b3fd9)